docs/install/requirements.rst

```diff
@@ -23,6 +23,7 @@ Envoy has the following requirements:
 * `c-ares <https://github.com/c-ares/c-ares>`_ (last tested with 1.13.0)
 * `backward <https://github.com/bombela/backward-cpp>`_ (last tested with 1.3)
 * `zlib <https://github.com/madler/zlib>`_ (last tested with 1.2.11)
+* `yaml-cpp <https://github.com/jbeder/yaml-cpp>`_ (last tested with sha e2818c423e5058a02f46ce2e519a82742a8ccac9).
```

Comment:
Unrelated to this change, but do you mind fixing Bazel to be 0.5.3? I just noticed this.